### ISSUE TITLE
[request] Docker image should use non-root user

### ISSUE THREAD

**Issue by or-shachar** [prometheus/cloudwatch_exporter#474]
# Proposal
I just noticed that our Dockerfile uses `eclipse-temurin:17-jre-focal` as runtime base image and never changes the user to be non-root.

Is this intentional? 

What if we create an "exporter" user and group and run it with limited permissions?

**Comment by matthiasr** [comment]
Indeed, running as root is not great. In the other exporter Docker images, we use user `65534` (`nobody`) – would that do here?

**Comment by dhoard** [comment]
@matthiasr PR https://github.com/prometheus/cloudwatch_exporter/pull/867

**Comment by dhoard** [comment]
Resolved in v0.18.0
